docs/source/getting-started/installation.rst

```diff
@@ -92,6 +94,20 @@ Note: if you encounter *Authorization Error (Error 400: invalid_request)* with t
 
 Hint: run ``az account subscription list`` to get a list of subscription IDs under your account.
 
+**Lambda**
```

Comment:
Lambda or Lambda Cloud?
```suggestion
**Lambda Cloud**
```

Reply:
I put Lambda because we refer it to it as 'Lambda' on the first page ('Lambda Cloud' felt too wordy). But a longer name might do better here.

Reply:
Actually, since it seems @concretevitamin prefers Lambda Cloud over Lambda, I'll change everything to Lambda Cloud.

Reply:
Updated!